src/components/PerrsonLink/PersonLink.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useLocation, useResolvedPath } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Partial<Person>;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, slug } = person;
+  const location = useLocation();
+  const parentPath = useResolvedPath('../').pathname;
+
+  return (
+    <Link
+      to={{
+        pathname: parentPath + slug,
+        search: location.search,
+      }}
+      className={classNames(
+        ({ 'has-text-danger': sex === 'f' }),
+      )}
```

Comment:
FIY: you can do the same like this:
```
className={classNames(sex === 'f' && 'has-text-danger')}
```
Both syntaxes if valid. But you can choose which you like better

Reply:
Your variant is more comprehensible, I guess